src/subscription_manager/managercli.py

```diff
@@ -195,8 +195,8 @@
     _("Entitlement Type:")
 ]
 
-SP_CONFLICT_MESSAGE = _("Due to a conflicting change made at the server the "
-                        "{attr} has not been set.\n{advice}")
+SP_CONFLICT_MESSAGE = _("Warning: A {attr} of \"{download_value}\" was recently set for this system "
+                        "by the entitlement server administrator.\n{advice}")
```

Comment:
Shouldn't this also be applied to the same process in syspurpose?

Reply:
Yeah, you are right.